nubis/puppet/files/squid-exporter/99-squid-exporter-startup

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
where is log from ?

Reply:
Fixed in e7978cccc52322594b559d0c43d98066f75aaef9